tests/cnf/ran/powermanagement/internal/tsparams/powermanagementvars.go

```diff
@@ -2,9 +2,25 @@ package tsparams
 
 import (
 	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/ranparam"
+	"github.com/openshift-kni/k8sreporter"
+	performancev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
+	corev1 "k8s.io/api/core/v1"
 )
 
 var (
 	// Labels represents the range of labels that can be used for test cases selection.
 	Labels = append(ranparam.Labels, LabelSuite)
+
+	// ReporterNamespacesToDump tells to the reporter which namespaces to collect pod logs from.
+	ReporterNamespacesToDump = map[string]string{
+		TestingNamespace: "",
+		PrivPodNamespace: "",
```

Comment:
I think this can be changed to the machine config NS as the perf profile change is applied via machine config daemon.